src/i18n/locales/en.json

```diff
@@ -182,6 +182,10 @@
 		"ProductQuantityLitre": "{0} L",
 		"UnknownProduct": "Unknown product name"
 	},
+	"ProductCategories": {
+		"Title": "Categories",
+		"Empty": "This product does not have any categories yet."
```

Comment:
Maybe:
"No categories have been added for this product yet." Or with extra "Help us @OFF page"